x-pack/plugins/maps/common/constants.ts

```diff
@@ -79,6 +81,22 @@ export enum SOURCE_TYPES {
   MVT_SINGLE_LAYER = 'MVT_SINGLE_LAYER',
 }
 
+export enum LICENSED_FEATURES {
+  GEO_SHAPE_AGGS = 'GEO_SHAPE_AGGS',
```

Comment:
Should `GEO_SHAPE_AGGS` be more specific and be something like `GEO_SHAPE_GEO_TILE_AGG` so we could track geo shape aggs independently?